packages/@lwc/template-compiler/README.md

```diff
@@ -11,7 +11,7 @@ yarn add --dev @lwc/template-compiler
 ## Usage
 
 ```js
-import compile from '@lwc/template-compiler';
+import { compile } from '@lwc/template-compiler';
```

Comment:
Named exports are way more compatible across the JS ecosystem than default exports. We should just recommend people to use this, even if we still support the default exports.